src/IssueViz.UnitTests/Editor/SelectedIssueTagging/Highlighting/IssueHighlightViewTaggerProviderTests.cs

```diff
@@ -48,7 +63,22 @@ protected override IViewTaggerProvider CreateTestSubject()
             aggregatorMock.Setup(x => x.CreateTagAggregator<ISelectedIssueLocationTag>(It.IsAny<ITextBuffer>()))
                 .Returns(Mock.Of<ITagAggregator<ISelectedIssueLocationTag>>());
 
-            return new IssueHighlightViewTaggerProvider(aggregatorMock.Object);
+            return new IssueHighlightViewTaggerProvider(aggregatorMock.Object, taggableBufferIndicator.Object);
+        }
+
+        [TestMethod]
+        public void CreateTagger_BufferIsNotTaggable_Null()
+        {
+            var textBuffer = TaggerTestHelper.ValidBuffer;
+            var textView = CreateValidTextView(textBuffer);
+
+            taggableBufferIndicator.Reset();
+            taggableBufferIndicator.Setup(x => x.IsTaggable(textBuffer)).Returns(false);
```

Comment:
This is not a good use of `TestInitialize`. The `taggableBufferIndicator` mock is only used in two tests, and in this one you are completely resetting it so the setup carried out in `TestInitialize` isn't used at all.

It would be better to have a `CreateTaggableIndicatorMock(bool isTaggable)` method that created and returned an appropriate object that each test could call.